From: Gunnar Hjalmarsson <gunnarhj@debian.org>
Subject: Change the default monospcace font for latin scipts

Upstream‘s default monospace font for latin scripts is currently
Noto Sans Mono, and that has been reflected in Debian since
fontconfig 2.14.1-3. Due to some deficiencies in that font, we change
it to the well established DejaVu Sans Mono. To cover situations when
the fonts-dejavu-mono package is not installed, we put the Noto Mono
font — included in the fonts-noto-mono package — as the second one in
the list.

Bug-Debian: https://bugs.debian.org/1028897
Forwarded: not-needed
Last-Update: 2023-08-20

--- a/conf.d/60-latin.conf
+++ b/conf.d/60-latin.conf
@@ -35,8 +35,9 @@
 	<alias>
 		<family>monospace</family>
 		<prefer>
-			<family>Noto Sans Mono</family>
 			<family>DejaVu Sans Mono</family>
+			<family>Noto Mono</family>
+			<family>Noto Sans Mono</family>
 			<family>Inconsolata</family>
 			<family>Andale Mono</family>
 			<family>Courier New</family>
